ios/flutter_radar.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  s.dependency 'RadarSDK', '3.1.5'
+  s.dependency 'RadarSDK', '3.5.9'
```

Comment:
@gabrielalao I think we need to bump `s.version` above

Reply:
ok

Reply:
fixed